Makefile

```
@@ -1,5 +1,6 @@
+VERSION ?= $(shell git describe --tags)
 # Image URL to use all building/pushing image targets
-IMG ?= controller:latest
+IMG ?= "controller:$(VERSION)"
```

This means if you override IMG you have to do Version yourself?
```suggestion
IMG_URL ?= "controller"
IMG ?= "$(IMG_URL):$(VERSION)"
```